package.json

```diff
@@ -12,7 +12,19 @@
   "scripts": {
     "build": "tsc",
     "prepare": "npm run build",
-    "test": "ava dist/test/**.js"
+    "test": "ava dist/test/**.js",
+    "coverage": "nyc ava dist/test/**.js --reporter=text-lcov"
```

Comment:
Looks like this reports avout 0% coverage on `.eslintrc.js` and generates a `coverage/` directory, perhaps needs `-x .eslintrc.js -x coverage`.